src/Plutus/Certification/Synchronizer.hs

```diff
@@ -87,9 +92,8 @@
 synchronizeDbTransactions :: (MonadIO m, MonadMask m) => [WalletTransaction] -> m ()
 synchronizeDbTransactions transactions = do
   -- filter out the outgoing transactions and sync them with the database
-  DB.withDb $ forM_ incomingTransactions storeTransaction
+  DB.withDb $ forM_ transactions storeTransaction
   where
```

Comment:
outgoing transaction are to be taken in consideration as well, because the system now will broadcast by itself transactions to solve overlapping addresses for different profiles